proj.py

```diff
@@ -0,0 +1,119 @@
+import argparse
+import matplotlib.pyplot as plt
+import numpy as np  # linear algebra
+import os
+import pandas as pd  # data processing, CSV file I/O (e.g. pd.read_csv)
+import seaborn as sns
+import sqlite3
+
+
+parser = ""
```

Comment:
Avoid contants but if you need constants, constants are always in the upper case according to pep-8.

Reply:
Updated all constants in accordance with pep-8. Also, updated few more style errors.